twake/frontend/src/app/scenes/App/ChannelsBar/ChannelsUser/ChannelsUser.js

```diff
@@ -211,8 +169,8 @@ export default class ChannelsUser extends Component {
             [],
             'Messages directs',
```

Comment:
Do we really need to pass french translation here ? @RomaricMourgues

Reply:
No, this is fallback translation in case the key is missing. It should be in english or removed (you can also remove the second argument empty array).
Languages.t("translation.key", ["variable1", "variable2"], "fallback if translation not found"); (the last two parameters are optional)